packages/myst-cli/src/config.ts

```diff
@@ -430,6 +430,7 @@ async function resolveProjectConfigPaths(
         }
       }),
     );
+    await session.loadPlugins(resolvedFields.plugins);
```

Comment:
We are explicitly waiting for the plugins to load prior to resolving the `parts` on the next line.